generateResources.py

```diff
@@ -980,6 +982,31 @@ def item(filename_parts, *layers, parent='item/generated'):
 # LEATHER / HIDES
 blockstate(('placed_hide',), 'tfc:hide_rack', {})
 
+# AGRICULTURE
+
+for crop in CROPS.keys():
```

Comment:
```suggestion
for crop, data in CROPS.items():
```
Rather than accessing each property by value